allvers/1999-02-02-GPL-2.0-from-gnu.org.txt.notes

```diff
@@ -0,0 +1,4 @@
+Links:
+    - http://web.archive.org/web/19990202012404/http://www.gnu.org/copyleft/gpl.txt
+Notes: same as the 1997-07-15-GPL-2.0-from-gnu-grep-copying
```

Comment:
So why would we include this one if it is exactly the same?

Reply:
Because this is the first time this one shows up at http://www.gnu.org/copyleft/gpl.txt

Reply:
It might have been there earlier, just not indexed earlier by the Internet Archive. It definitely has shown up before in archives of GNU software. I guess that having it as a data point ("first time it appears in the Internet Archive") makes sense.